lib/active_merchant/billing/gateways/bambora.rb

```diff
@@ -0,0 +1,179 @@
+require "nokogiri"
+
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    class BamboraGateway < Gateway
+      self.live_url = 'https://www.bambora.co.nz/interface/api/dts.asmx'
+      self.test_url = 'https://demo.bambora.co.nz/interface/api/dts.asmx'
```

Comment:
Are the url's country specific? https://dev-apac.bambora.com/checkout/guides/get-up-and-running/migration-to-production

Reply:
This was not spotted in the internal reviews, but I'm confirming with a technical contact that it will support both countries. Looking closer, the correct URL should be  https://www.bambora.co.nz/access/index.aspx